src/coreclr/nativeaot/Runtime/inc/MethodTable.h

```diff
@@ -260,22 +260,6 @@ class MethodTable
         return (m_uFlags & OptionalFieldsFlag) != 0;
     }
 
-    bool IsEquivalentTo(MethodTable * pOtherEEType)
```

Comment:
@MichalStrehovsky Applies https://github.com/dotnet/runtime/pull/91921 in the unmanaged runtime.